waspc/src/Wasp/Generator/WebAppGenerator.hs

```diff
@@ -106,14 +107,26 @@ npmDepsForWasp _spec =
             ("@tanstack/react-query", "^4.13.0"),
             ("react-router-dom", "^5.3.3"),
             ("react-scripts", "5.0.1")
-          ],
+          ]
+          ++ depsRequiredByTailwind spec,
       -- NOTE: In order to follow Create React App conventions, do not place any dependencies under devDependencies.
       -- See discussion here for more: https://github.com/wasp-lang/wasp/pull/621
       N.waspDevDependencies =
         AS.Dependency.fromList
           []
     }
 
+depsRequiredByTailwind :: AppSpec -> [AS.Dependency.Dependency]
```

Comment:
We could, but we'd have to tell them to add it as a step and also add checks for them. It could also make things harder over time as versions skew between our stuff and theirs, or if default config files change we give examples to, etc. I think it is pretty safe for us to provide them to the users atm, but open to counterarguments. 🤷🏻‍♂️

Reply:
Sounds good for the start and then we can see!